### DISCUSSION TITLE
New Teleport event in the API

### DISCUSSION THREAD

**Discussion by MrLarkyy** [BG-Software-LLC/SuperiorSkyblock2#398] [Pending Feature Requests]
Hello, could you please add an event that is called before a player is teleported to player's/his island/warp ? I will really need that, because I am just using Passengers and when a player got a passenger on himself, he cannot be teleported... So I just made a system to remove the player's passenger for 1 tick and then set it back, but I cannot do it with SuperiorSkyblock as there's no event for that... :/ Would really appreciate that.

Thank you, Larkyy.

**Comment by OmerBenGera** [comment]
PlayerTeleportEvent?

**Reply by MrLarkyy** [reply]
Already tried and it's not called when you got passengers.

**Reply by MrLarkyy** [reply]
![image](https://user-images.githubusercontent.com/48769171/126882278-8d7c9b8e-8fff-403a-805c-7a382d877a3e.png)
